wagtail/contrib/search_promotions/views.py

```diff
@@ -1,85 +1,64 @@
 from django.core.paginator import InvalidPage, Paginator
 from django.db import transaction
-from django.db.models import Sum, functions
+from django.db.models import F, Sum, functions
 from django.http import Http404
 from django.shortcuts import get_object_or_404, redirect
 from django.template.response import TemplateResponse
 from django.urls import reverse
 from django.utils.translation import gettext as _
```

Comment:
Be careful now that we have translatable strings at the class definition level (not inside a function). These strings will need to use `gettext_lazy`, so that they're not immediately translated at application startup.